modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/exportimport/data/handler/test/AccountEntriesAdminPortletDataHandlerTest.java

```diff
@@ -139,21 +159,59 @@ public void testExportImportAccountEntries() throws Exception {
 							HashMapBuilder.put(
 								PortletDataHandlerKeys.PORTLET_DATA,
 								new String[] {Boolean.TRUE.toString()}
-							).build())),
-			larFile);
+							).build()));
+
+		_exportImportLocalService.importLayouts(
+			exportImportConfiguration, larFile);
 
 		Assert.assertNotNull(
 			_accountEntryLocalService.fetchAccountEntryByExternalReferenceCode(
 				accountEntry.getExternalReferenceCode(),
 				TestPropsValues.getCompanyId()));
 
+		accountGroup =
+			_accountGroupLocalService.fetchAccountGroupByExternalReferenceCode(
+				accountGroup.getExternalReferenceCode(),
+				TestPropsValues.getCompanyId());
+
+		Assert.assertEquals(
+			WorkflowConstants.STATUS_INCOMPLETE, accountGroup.getStatus());
+
 		organization =
 			_organizationLocalService.fetchOrganizationByExternalReferenceCode(
 				organization.getExternalReferenceCode(),
 				TestPropsValues.getCompanyId());
 
 		Assert.assertEquals(
 			WorkflowConstants.STATUS_INCOMPLETE, organization.getStatus());
+
+		List<ExportImportReportEntry> exportImportReportEntries =
+			_exportImportReportEntryLocalService.getExportImportReportEntries(
+				TestPropsValues.getCompanyId(),
+				exportImportConfiguration.getExportImportConfigurationId());
+
+		Assert.assertEquals(
+			exportImportReportEntries.toString(), 2,
+			exportImportReportEntries.size());
+
+		ExportImportReportEntry exportImportReportEntry =
+			exportImportReportEntries.get(1);
+
+		Assert.assertEquals(
+			accountGroup.getExternalReferenceCode(),
+			exportImportReportEntry.getClassExternalReferenceCode());
+		Assert.assertEquals(
+			ExportImportReportEntryConstants.TYPE_INCOMPLETE,
+			exportImportReportEntry.getType());
+
+		exportImportReportEntry = exportImportReportEntries.get(0);
+
+		Assert.assertEquals(
+			organization.getExternalReferenceCode(),
+			exportImportReportEntry.getClassExternalReferenceCode());
+		Assert.assertEquals(
+			ExportImportReportEntryConstants.TYPE_INCOMPLETE,
+			exportImportReportEntry.getType());
```

Comment:
I would invert these .get(0) should go before .get(1)

Reply:
or better, use a filter
so we can do
```
import
check account entry not null
check export size
check account group status
check export contains the account group with status incomplete
check organization status
check export contains the organization with status incomplete
```

Reply:
I think we did a filter like this in the accountresourcetest when we check for the permissions